content/300-guides/200-deployment/110-deployment-guides/500-deploying-to-netlify.mdx

```diff
@@ -6,68 +6,87 @@ metaDescription: 'Learn how to deploy Node.js and TypeScript applications that a
+
+With an empty database in place, you continue with the Prisma workflow that involves pushing the data model defined in the `schema.prisma` file to the database and then generating Prisma Client.
 
-This guide starts with an empty database that you must have already created and looks as follows:
+The steps are outlined below.
 
-1. Migrate the existing Prisma schema to your existing database using `prisma db push`.
+1. Push the existing Prisma schema to your existing database using `prisma db push`.
 2. Run `prisma generate` to generate the Prisma Client based on the Prisma schema.
 
 ## 1. Fork the <inlinecode>deployment-example-netlify</inlinecode> repository
 
-Go to [`deployment-example-netlify`](https://github.com/prisma/deployment-example-netlify) and fork the repository by clicking the **Fork** button in the top-right corner:
+To begin, fork the repository of the deployment example project [`deployment-example-netlify`](https://github.com/prisma/deployment-example-netlify).
+
+With a forked repository, you create a copy of the repository to which you will be able to commit your own changes. Later, you will connect the forked repository to Netlify so that changes are automatically built and deployed.
+
+### Steps
+
+1. Go to [`deployment-example-netlify`](https://github.com/prisma/deployment-example-netlify) and click the **Fork** button in the top-right corner to fork the repository:
+
+   ![GitHub repository - click Fork button](images/500-02-deploying-to-netlify-example-repo-click-fork.png)
 
-![fork button](https://res.cloudinary.com/prismaio/image/upload/v1628765101/docs/nyap5pW_g0tbuw.png)
+2. On the **Create a new fork** page, configure the forked repository.
+   1. (Optional) From **Owner**, select a GitHub account or organization as the owner.
+   2. (Optional) In **Repository name**, modify the repository name if necessary.
+   3. Leave as selected **Copy the `main` branch only**.
+3. Click **Create fork**.
+   ![GitHub - Create a new fork page](images/500-03-deploying-to-netlify-example-repo-create-fork-page.png)
 
-By forking, you create a copy of the repository to which you will be able to commit your changes. Later in the guide, you will connect the forked repository to Netlify so that changes are automatically built and deployed.
+### Result
 
-**Checkpoint:** After the forking is complete, you should see the forked repository in your account, for example `https://github.com/YOUR_GITHUB_USERNAME/deployment-example-netlify`
+After the forking is complete, you should see the forked repository in your account, for example `https://github.com/YOUR_GITHUB_USERNAME/deployment-example-netlify`.
 
 ## 2. Clone the fork and install dependencies
 
-Open your terminal, navigate to a location of your choice and clone your fork of the `deployment-example-netlify`:
+You need to clone the repository to a folder on your local filesystem to install all required npm packages.
```

Comment:
It looks like *filesystem* and *file system* are both accepted forms. We should standardize on one. I'll add it to style guide questions.